src/citrine/informatics/scores.py

```diff
@@ -67,8 +67,8 @@ class LIScore(Serializable['LIScore'], Score):
     constraints = properties.List(properties.Object(Constraint), 'constraints')
     typ = properties.String('type', default='MLI')
 
-    def __init__(self, *,
-                 name: Optional[str] = None,
+    def __init__(self,
+                 name: Optional[str] = None, *,
```

Comment:
`name` is actually deprecated for scores, so I don't think we should privilege it in this way